### PR TITLE
Divya hr

### PR DESCRIPTION
**Pull request by Samrudhsaish**
Hi, I am working on Human Management Resource module.

### CODE REVIEW

File: .idea/.gitignore
Comment (rambabu) [inline .idea/.gitignore:1]: add a gitignore file

File: src/com/acro/emp/hrmodule/HR.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package com.acro.emp.hrmodule;
+
+public class HR {
+  public static void main(String[] args){
+
+
```
Comment (rambabu) [inline src/com/acro/emp/hrmodule/HR.java]: I need props and methods

**Review by rambabu — CHANGES_REQUESTED** [review]

File: src/com/acro/emp/AddProject.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+    public String checkingAddedProject() {
+        if (projectName == " ") {
+            System.out.println("Project name not  mentioned");
+        } else {
+            System.out.println("Project name  mentioned");
+        }
+        return projectName;
+    }
+        public static void main (String[]args){
```
Comment (Ranianirudh) [inline src/com/acro/emp/AddProject.java:28]: main is not needed in all the classes.

File: src/com/acro/emp/AddProject.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
+package com.acro.emp;
+
+public class AddProject extends ProjectDetails{
```
Comment (Ranianirudh) [inline src/com/acro/emp/AddProject.java:3]: As for as I know, Inheritance is not required here.

**Review by Ranianirudh — CHANGES_REQUESTED** [review]

**Pull request merged**